src/modules/rewards/components/RewardsActivitiesListItem.tsx

```diff
@@ -124,13 +140,24 @@ const RewardsActivitiesListItem: FC<RewardActivitiesListItemProps> = ({
                     isLoading={isLoading}
                   />
 
-                  {!!activity.expiryType && (
-                    <Box display="flex">
+                  <Box
+                    display="flex"
+                    gap="spacing-xxs"
+                  >
+                    {!!activity.expiryType && (
```

Comment:
```suggestion
                    {!!activity?.expiryType && (
```

Reply:
@mishramonalisha76 expiryType will be always available for every activity.